src/View/Helper/LayoutHelper.php

```diff
@@ -95,6 +95,27 @@ public function messages(): bool
         return $this->_View->getName() != 'Login';
     }
 
+    /**
+     * Return title with object title and current module name
+     *
+     * @return string title with object title and current module name separated by '|'
+     */
+    public function title(): string
+    {
+        $currentModule = (array)$this->getView()->get('currentModule');
+        $object = (array)$this->getView()->get('object');
+        $title = '';
+        if (!empty($currentModule) && !empty($currentModule['name'])) {
+            $currentModuleName = $currentModule['name'];
+            if (!empty($object) && !empty($object['attributes']['title'])) {
```

Comment:
```suggestion
            if (Hash::check($object, 'attributes.title')) {
```
You can use `Cake\Utility\Hash::check` in here